packages/hash/api/src/auth/ory-kratos.ts

```diff
@@ -0,0 +1,10 @@
+import { Configuration } from "@ory/client";
+import { V0alpha2Api as OpenSourceV0alpha2Api } from "@ory/kratos-client";
+import { getRequiredEnv } from "../util";
+
+/** @todo: connect to admin endpoint when admin methods are required */
+const publicUrl = getRequiredEnv("ORY_KATROS_PUBLIC_URL");
```

Comment:
Kratos not kartos

Reply:
Addressed in https://github.com/hashintel/hash/pull/889/commits/0d309de72bd8cecc79a5b516f02630f4c7d4a33b